### PR TITLE
Refactoring

### LINKED ISSUE #41: Tidy up main.cpp
Remove functions from the top (above `setup`) of  `main.cpp` and put them in their correct files.

### PR DESCRIPTION
**Pull request by alphaville**
## Main Changes

- average initial attitude computed in ahrs (see `averageQuaternion`)
- new method  in motor driver
- new method  in radio
- `logSerial`, `setupBuzzer`, `buzz` and `waitForPiSerial` go to util
- delete `config.cpp`
- change all `Serial.print` to `logSerial`
- deactivate all printing while flying
- rename `BZZZ_VERBOSITY` to `BZZZ_LOGGING_LEVEL`

## Associated Issues

- Closes #41

## DevOps

Before we merge this:

- We should merge #44 into `dev` first
- Then merge `dev` into `feature/41-tidy-up-main`
- And then move `initAngularVelocity` to AHRS (as we've done with `averageQuaternion` here)

### CODE REVIEW

**Comment by alphaville** [comment]
@jamie-54 can you have a look and make sure that everything makes sense and that it works on the quadcopter? I haven't really changed any functionality; I just reorganised the code.

**Comment by jamie-54** [comment]
Hi looks good and everything makes sense I'll test it on the hardware tomorrow just to be sure.

Just wondering do you want to keep `setGainsFromRcTrimmers` in the `main.cpp` or move it somewhere else? 
I get that it doesn't really have a clear file that it belongs to.

**Comment by jamie-54** [comment]
> @jamie-54 this actually is part of the `loop`. I just moved this functionality to `setGainsFromRcTrimmers` for `loop` to be more readable. I don't think it belongs to another file.

Oh yea I see what's happening now, looks good

File: src/ahrs.cpp [hunk 1]
```diff
@@ -66,6 +67,79 @@ namespace bzzz
         return m_imu.update();
     }
 
+    /**
+     * At the beginning the AHRS hasn't converged, so we need to discard
+     * some measurements
+     */
+    void AHRS::discardImuMeasurements(size_t numMeasurements)
```
Comment (jamie-54) [inline src/ahrs.cpp:74]: Is this documentation needed here since it's in the `ahrs.hpp`?

**Comment by alphaville** [comment]
@jamie-54 I'm merging this now, but can you check whether everything still works (it should). I've made some very minor changes (see last commit).

**Comment by jamie-54** [comment]
> @jamie-54 I'm merging this now, but can you check whether everything still works (it should). I've made some very minor changes (see last commit).

No problem

**Pull request merged**